detect_gitlab.go

```diff
@@ -0,0 +1,47 @@
+package getter
+
+import (
+	"fmt"
+	"net/url"
+	"strings"
+)
+
+// GitHubDetector implements Detector to detect GitHub URLs and turn
```

Comment:
Please update comment to replace GitHub with Gitlab

Reply:
Done